packages/labs/src/controllers/async-task.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {ReactiveElement, notEqual} from 'reactive-element';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type TaskFunction = (...args: Array<any>) => any;
```

Comment:
It would be great to offer a way to abort the previous pending task if there is one. This would involve letting the task function optionally get an AbortSignal, separate from the deps. For that and other future evolution, we might want the task function signature to be `(args: Array<unknown>, opts?: TaskOptions) => Promise`.

Reply:
Changed task arguments to an array to facilitate future changes.

Reply:
I think this signature needs to change too, to: `export type TaskFunction = (args: Array<any>) => any;`.

Also, can the `any`s change to `unknown` here and below?